app/page.tsx

```
@@ -46,7 +55,11 @@ export default function Home() {
     }
   }
 
-  const handleContinueClick = () => {
+  const handleContinueClick = ({}: {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)